translations/harbour-determinant-es.ts

```diff
@@ -267,11 +267,11 @@
     </message>
     <message>
         <source>Invitation</source>
-        <translation type="unfinished"></translation>
+        <translation>Invitación </translation>
     </message>
     <message>
         <source>Logout</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerrar sesión </translation>
```

Comment:
trailing whitespace

Reply:
What wrong here? you mean without space between words?

Reply:
I mean the space at the end. It should be removed.